ee/hogai/assistant/base.py

```diff
@@ -32,8 +32,7 @@
 from posthog.models import Team, User
 from posthog.sync import database_sync_to_async
 
-from ee.hogai.graph.base import BaseAssistantNode
-from ee.hogai.graph.graph import AssistantCompiledStateGraph
+from ee.hogai.graph.base import AssistantCompiledStateGraph, BaseAssistantNode
```

Comment:
FYI, `AssistantCompiledStateGraph` is gone on my `loop-executor-ui-full` branch, so there will be conflicts (I can fix them once we get there)